src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -232,6 +233,8 @@ private String tableToSnsTopic(String table, Operation operation) {
         default -> null;
       };
       case INSERT, LOAD, UPDATE -> switch (table) {
+        case TABLE_CONDITIONS_OF_JOINING ->
```

Comment:
there is no CoJ delete mapping, because there is no mechanism for this to happen at this point 🤔